README.md

```diff
@@ -28,7 +28,7 @@ This group fansite must have the following HTML files:
 
 - `facts.html` should include a series of fun facts or more detailed information about your topic.
 
-- `gallery.html` must include at least 3 images with descriptions and links to where the images came from if applicable. 
+- `gallery.html` must include at l`east 3 images with descriptions and links to where the images came from if applicable. 
```

Comment:
It's important to run `git status` before committing changes so we can see what files have been changed and revert changes like this that we don't intend to commit.